WORKSPACE

```diff
@@ -72,3 +72,14 @@ envoy_dependencies()
 load("@envoy//bazel:dependency_imports.bzl", "envoy_dependency_imports")
 
 envoy_dependency_imports()
+
+ABSL_SHA = "61c9bf3e3e1c28a4aa6d7f1be4b37fd473bb5529"
+
+ABSL_SHA256 = "7ddf863ddced6fa5bf7304103f9c7aa619c20a2fcf84475512c8d3834b9d14fa"
+
+http_archive(
```

Comment:
absl is already loaded by dependencies above, linking different versions of absl is likely causing problems.

Reply:
Thanks.. Removed